marketpulse/geo/lookup.py

```diff
@@ -0,0 +1,36 @@
+from urlparse import urljoin
+
+from django.conf import settings
+
+from django_countries import countries
+import requests
+
+
+COUNTRY_CODES = {key: value for (value, key) in list(countries)}
+
+
+def reverse_geocode(lat, lng):
+    """Query Mapbox API to get data for lat, lng"""
+
+    query = '{0},{1}.json'.format(lng, lat)
+    url = urljoin(settings.MAPBOX_GEOCODE_URL, query)
+    params = {'access_token': settings.MAPBOX_TOKEN}
+    response = requests.get(url, params=params)
+
+    if response.status_code != 200:
+        raise Exception('{0} on: {1}'.format(response.status_code, url))
```

Comment:
Since we are going to use this in the save method, maybe we should fail silently.